### ISSUE TITLE
[gradle] JReleaser throws a NumberFormatException when using a Gradle RC build

### ISSUE THREAD

**Issue by bitspittle** [jreleaser/jreleaser#1051]
## Background

I'm running into an issue where Gradle is crashing with Kotlin 1.7.20, and the final stable version isn't out yet. Instead, I'm trying to use the latest RC to see if that works. See also: https://github.com/gradle/gradle/issues/22320

After trying to do this, JReleaser crashes unfortunately.

## Repro

Seen in both 1.0.0 and 1.3.1

* In your project's `gradle/wrapper/gradle-wrapper.properties` file...
* Set `distributionUrl=https\://services.gradle.org/distributions/gradle-7.6-rc-3-bin.zip`

### Result

```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':kobweb:cli:kobweb'.
> For input string: "6-rc-3"

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Exception is:
org.gradle.api.ProjectConfigurationException: A problem occurred configuring project ':kobweb:cli:kobweb'.
	at (a ton of gradle callstack lines elided)
Caused by: java.lang.NumberFormatException: For input string: "6-rc-3"
	at org.jreleaser.gradle.plugin.internal.JReleaserProjectConfigurer$_configureModel_closure1.doCall(JReleaserProjectConfigurer.groovy:363)
	at org.jreleaser.gradle.plugin.internal.JReleaserProjectConfigurer$_configureModel_closure1.call(JReleaserProjectConfigurer.groovy)
	at org.jreleaser.gradle.plugin.internal.JReleaserProjectConfigurer.configureModel(JReleaserProjectConfigurer.groovy:362)
	at org.jreleaser.gradle.plugin.internal.JReleaserProjectConfigurer.configure(JReleaserProjectConfigurer.groovy:73)
	at org.jreleaser.gradle.plugin.JReleaserPlugin.configureJReleaser(JReleaserPlugin.groovy:69)
	at org.jreleaser.gradle.plugin.JReleaserPlugin.access$0(JReleaserPlugin.groovy)
	at org.jreleaser.gradle.plugin.JReleaserPlugin$1.execute(JReleaserPlugin.groovy:62)
	at org.jreleaser.gradle.plugin.JReleaserPlugin$1.execute(JReleaserPlugin.groovy)
	at (a ton of gradle callstacks elided) 
```

### Environment Information

- **Operating System**: Linux
- **JReleaser Version:** 1.0.0 and 1.3.1
- **JDK Version:** JDK 11

trace.log is empty.

**Comment by aalmiray** [comment]
Affected code is https://github.com/jreleaser/jreleaser/blob/257c2e76d5d02e158f6ccfb4205d95fbc87154a9/plugins/jreleaser-gradle-plugin/src/main/groovy/org/jreleaser/gradle/plugin/internal/JReleaserProjectConfigurer.groovy#L362-L363

**Comment by aalmiray** [comment]
Released in v1.4.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.4.0
